src/components/Dashboard/Analytics/MainBarChartData.json

```diff
@@ -0,0 +1,48 @@
+{
```

Comment:
You can combine these three json files in one file

Reply:
Yes, you can leave it in this way. I was only worried that when the developer would like to move it to the backend that he/she will think that every file needs to have a different structure when this is incorrect because the files have similar obj properties.